### PR TITLE
fix: Explore style changes based on design updates

### PR DESCRIPTION
**Pull request by kayleegeorge**
***KEEP DRAFT UNTIL DESIGN SETTLES MORE -- NOT DONE YET

Update color nits based on Explore theme design updates

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -82,7 +82,7 @@
         pointer-events: none;
         width: 200vw;
         height: 200vh;
-        background: radial-gradient(50% 50% at 50% 50%, #fc077d10 0%, rgba(255, 255, 255, 0) 100%);
+        background: linear-gradient(180deg, #202738 0%, #070816 100%);
```
Comment (kayleegeorge) [inline public/index.html:85]: @vm not sure how to feature flag this one

File: src/components/Charts/circleCorners.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
+/* THIS FILE IS FROM A NON-TYPESCRIPT LIBRARY THAT WOULDN'T IMPORT PROPERLY
```
Comment (vm) [inline src/components/Charts/circleCorners.js:1]: delete

File: public/index.html (hunk 1 above)
Comment (vm) [reply public/index.html]: you can't

**Comment by kayleegeorge** [comment]
Migrated to here: https://github.com/Uniswap/interface/pull/4294

**Pull request closed without merging**